Source/relay/TourGuide/Paths/Avatar of Shadows over Loathing.ash

```diff
@@ -23,3 +23,63 @@ void PathShadowsOverLoathingGenerateTasks(ChecklistEntry [int] task_entries, Che
         task_entries.listAppend(ChecklistEntryMake("__skill Motif", "", ChecklistSubentryMake(main_title, "", description), -11));
     }
 }
+
+record CursedItem {
```

Comment:
this record format works pretty well IMO; on first read it felt maybe a tiny bit overengineered but i think it works nicely and associates a lot of different info well.

Reply:
In TS/JS I would have made these plain objects but ASH maps work very oddly IMO and it seemed clearer to just use records here.